lsstsal/scripts/gencmdaliascode.tcl

```diff
@@ -30,12 +30,43 @@ global CMD_ALIASES CMDS DONE_CMDEVT ACKREVCODE REVCODE SAL_WORK_DIR OPTIONS
      foreach i $CMD_ALIASES($subsys) { 
        if { [info exists CMDS($subsys,$i,param)] } {
          puts $fout "
+/** Issue the [set i] command to the SALData subsystem
```

Comment:
In the context of ts_xml, what would an example of int issueCommand_[set i]( SALData_command_[set i]C *data ); look like?

If we are using ATTCS as an example I would guess `int issueCommand_ATTCS_command_target`

Giving an example like this just for the first appearance of this set would give me a better idea of what the rest does.

Reply:
With reference to the XML ATTCS_<EFDB_Topic>ATTCS_command_start</EFDB_Topic> would lead to the generation of an
issueCommand_start method the SAL_ATTCS object. The ATTCS_command_startC structure contains the payload ie the values of <item>'s for the command